addon/ng2/models/webpack-build-common.ts

```diff
@@ -4,8 +4,12 @@ import * as HtmlWebpackPlugin from 'html-webpack-plugin';
 import * as webpack from 'webpack';
 import { ForkCheckerPlugin } from 'awesome-typescript-loader';
 import { CliConfig } from './config';
+import { ServeTaskOptions } from '../commands/serve';
```

Comment:
Is this a used import? Otherwise should be removed. You should run `npm run lint` to help you catch these things in your project.